src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -6,6 +6,6 @@ public class ColorSupplier {
     private Random random = new Random();
 
     public String getRandomColor() {
-        return Color.values()[random.nextInt(7)].toString();
+        return Color.values()[random.nextInt(Color.values().length)].toString();
```

Comment:
you should've use name() method instead of toString() as it was mentioned in checklist, I attached a screenshot the last time